src/features/components/templates/navigation/tab-nav-without-border/index.tsx

```diff
@@ -80,22 +84,24 @@ const NavigationTabWithoutBorder = ({ tab_data }: NavigationTabWithoutBorderType
                 justify="around"
                 md={{ justify: 'center', padding: '10x' }}
                 visible="larger-than-phone"
+                className="wrapper_navigation"
             >
-                {tab_data.map((tab_item) => (
-                    <Flex.Box direction="col" key={tab_item.option_name}>
-                        <NavigationTabMenu
-                            key={tab_item.option_name}
-                            tab_items={tab_item}
-                            selected={isActiveLink(tab_item.to, tab_item?.active_path)}
-                            is_no_border_bottom
-                            icon={
-                                isActiveLink(tab_item.to, tab_item?.active_path)
-                                    ? `${tab_item.selected_src}#${tab_item.option_name}`
-                                    : `${tab_item.src}#${tab_item.option_name}`
-                            }
-                        />
-                    </Flex.Box>
-                ))}
+                {!tabs_loading &&
+                    tab_data.map((tab_item) => {
+                        return (
+                            <NavigationTabMenu
+                                key={tab_item.option_name}
+                                tab_items={tab_item}
+                                selected={tab_item.option_name === selected_tab_name}
+                                is_no_border_bottom
+                                icon={
+                                    tab_item.option_name === selected_tab_name
+                                        ? `${tab_item.selected_src}#${tab_item.option_name}`
+                                        : `${tab_item.src}#${tab_item.option_name}`
```

Comment:
can we simplify this by adding this inside a const and call here ?

Reply:
hi @aswathy-deriv  this is inside a map function we cannot pull this condition outside , since we are accessing the **tab_item.option_name**  this